docs/tasks/issuers/setup-acme/dns01/index.rst

```diff
@@ -64,6 +64,26 @@ Example usage::
 
 .. _supported-dns01-providers:
 
+Delegated Domains for DNS01
+===========================
+
+By default, cert-manager will not follow CNAME records pointing to subdomains.
+
+If granting cert-manager access to the root DNS zone is not desired, then the
+_acme-challenge.example.com subdomain can instead be delegated to some other,
+less privileged domain.
+This is done by adding an additional entry in `solvers`:
+
+.. code-block:: yaml
+
+   solvers:
```

Comment:
Can you include the path leading up to this in the example? e.g.:

```yaml
apiVersion: certmanager.k8s.io/v1alpha1
kind: Issuer
metadata:
  ...
spec:
  acme:
    ...
    solvers:
    - dns01:
        cnameStrategy: Follow
        cloudflare:
          ...
```

You can also use emphasize-lines to highlight the `cnameStrategy` line 😄